.gitignore

```diff
@@ -67,9 +67,20 @@ SwiftScripts/.build-**
 # Local Netlify folder
 .netlify
 
+# Generated js files we don't need committed
+Sources/ApolloCodegenLib/Frontend/dist/
+
 # Local generated packages we don't need in the main project
 Sources/AnimalKingdomAPI/Generated/Package.swift
 Sources/AnimalKingdomAPI/Generated/Package.resolved
-
-# Generated js files we don't need committed
-Sources/ApolloCodegenLib/Frontend/dist/
+Tests/TestCodeGenConfigurations/**/AnimalKingdomAPI/
+Tests/TestCodeGenConfigurations/**/AnimalKingdomAPITestMocks
+Tests/TestCodeGenConfigurations/**/Package.resolved
+Tests/TestCodeGenConfigurations/**/Podfile.lock
+Tests/TestCodeGenConfigurations/**/TestMocks
+Tests/TestCodeGenConfigurations/EmbeddedInTarget-RelativeAbsolute/PackageOne/**/*.graphql.swift
+Tests/TestCodeGenConfigurations/EmbeddedInTarget-RelativeAbsolute/PackageTwo/Sources/PackageTwo
+!Tests/TestCodeGenConfigurations/Other-CustomTarget/AnimalKingdomAPI/AnimalKingdomAPI.h
```

Comment:
The `!` files are ones that we specifically want to keep because they either configure the cache keys for the tests to pass of they're project files that are not generated but required to build.